modules/ROOT/pages/download-and-install-service-mesh.adoc

```diff
@@ -81,5 +81,27 @@ Anypoint Service Mesh is installed and the status of all pods is displayed:
 +
 image::verified-service-mesh-installer-output.png[]
 
+=== Configure Headers for Client Authorization-based Policies
```

Comment:
When used in titles or headings, capitalize the second word of a hyphenated compound exactly as you would if the word stood alone:
```suggestion
=== Configure Headers for Client Authorization-Based Policies
```

Reply:
If this topic is about whitelisting, shouldn't the title use the term?

Reply:
Good point! Done! Thanks!